src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -194,6 +194,13 @@ <h3>{{ 'artemisApp.courseOverview.exerciseDetails.problemStatement' | artemisTra
                     </div>
                 </div>
             </ng-template>
+            <jhi-reset-repo-button
+                *ngIf="exercise.type === PROGRAMMING && exercise.studentParticipations && exercise.studentParticipations.length"
```

Comment:
🧼 would prefer a function in `.component.ts` as it is more verbose and gives us type checking. Using falsyness is kind of dirty

Reply:
We try to avoid calling functions in the html template